### PR TITLE
created team members page and popup

### PR DESCRIPTION
**Pull request by NoumanEjaz-dev**

### CODE REVIEW

File: dev.next.config.js.sample
Comment (keyskull) [inline dev.next.config.js.sample:1]: please keep the `dev.next.config.js.sample` file in the branch

File: components/component/team-members-page-view.tsx [hunk 1]
```diff
@@ -0,0 +1,228 @@
+import React, { useState } from 'react';
+import { Button, Checkbox, Modal, Table } from 'flowbite-react';
```
Comment (keyskull) [inline components/component/team-members-page-view.tsx]: I think next-ui covered all these components. If it is possible, please replace from the next-ui

**Review by keyskull — CHANGES_REQUESTED** [review]

File: dev.next.config.js.sample
Comment (NoumanEjaz-dev) [reply dev.next.config.js.sample:1]: Updated the file

File: components/component/team-members-page-view.tsx (hunk 1 above)
Comment (NoumanEjaz-dev) [reply components/component/team-members-page-view.tsx]: Replaced these to the next-ui components

**Review by keyskull — APPROVED** [review]

**Pull request merged**